yoti-sdk-api/src/main/java/com/yoti/api/client/docs/session/instructions/document/SelectedDocument.java

```diff
@@ -0,0 +1,49 @@
+package com.yoti.api.client.docs.session.instructions.document;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public abstract class SelectedDocument {
+
+    @JsonProperty("type")
+    private final String type;
+
+    @JsonProperty("country_code")
+    private final String countryCode;
```

Comment:
Although these are identical for the Id + Supplementary Docs, I have a suspicion that one day we'll come to regret defining them here, somehow.